js/init.js

```diff
@@ -26,22 +27,24 @@ function init_map(){
 				color : '#000',
 				weight : 2
 			};
-    		},
-    		onEachFeature : function( feature, layer ) {
-	    		layer.on({
+    	},
+    	onEachFeature : function( feature, layer ) {
+	    	layer.on({
 				mouseover : highlightFeature,
 				mouseout : resetHighlight,
 				click : featureClick
 			});
-    		}
+    	}
 	});
 		
-	neighborhoods = omnivore.topojson( endpoint + "/topojson/neighborhoods/id%2Cname/", null, layerStyle )
+	neighborhoods = omnivore.topojson( endpoint + "/topojson/neighborhoods/id%2Cname%2Carea/", null, layerStyle )
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'omnivore' is not defined.